### DISCUSSION TITLE
BTCV tutorial error with data loading

### DISCUSSION THREAD

**Discussion by hreso110100** [Project-MONAI/tutorials#1241] [Q&A]
When I am loading BTCV data I got this error:

**INFO:nibabel.global:pixdim[0] (qfac) should be 1 (default) or -1; setting qfac to 1
pixdim[0] (qfac) should be 1 (default) or -1; setting qfac to 1**

**To Reproduce**
1. Load data with dataloader and dataset used in BTCV UNETR 3D tutorial
2. Observe if error occured and data loaded properly

**Expected behavior**

Data should be loaded without errors.

**Environment (please complete the following information):**
 - Ubuntu
 - Python 3.8
 - MONAI 0.9.0
 - CUDA/cuDNN 11.4
 - GPU - RTX 2080 8GB

**Comment by wyli** [comment]
that message is at the `INFO` level, I don't think it stops the python program. could you please post the full log?
